middlewares/authzWorkspaceCookieCheck.js

```diff
@@ -0,0 +1,25 @@
+const asyncHandler = require('express-async-handler');
+
+const csrf = require('../lib/csrf');
+const config = require('../lib/config');
+
+module.exports = asyncHandler(async (req, res, next) => {
```

Comment:
Nit: nothing async here, so this isn't strictly necessary.

Reply:
Removed.